### PR TITLE
254838: Updates to infra helm chart and appConfig

### PR DESCRIPTION
**Pull request by jeevankuduvaravindran**
Updates to infra helm chart and appConfig

[AB#254838](https://dev.azure.com/defragovuk/93c65135-d16b-49d6-a191-4a5313532779/_workitems/edit/254838)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-demo-payment-service",
-  "version": "4.2.11",
+  "version": "4.2.14",
```
Comment (kbabigumira) [inline package.json:3]: skipped 4.2.12

Comment (jeevankuduvaravindran) [reply package.json]: @kbabigumira fixed

**Review by DaRoszko — APPROVED** [review]

**Review by ganeshg2491 — DISMISSED** [review]

File: helm/ffc-demo-payment-service/values.yaml [hunk 2]
```diff
@@ -64,19 +64,15 @@ serviceAccount:
...
     labelFilter: 'ffc-demo-payment-service'
     refresh:
       interval: 2m
       monitorKey: 'Sentinel'
       monitorLabel: 'ffc-demo-payment-service'
-
-managedIdPrefix: sndadpinfmid1401
 userAssignedIdentity:
-  managedIdName: ffc-demo-services
+  managedIdName:
```
Comment (kbabigumira) [inline helm/ffc-demo-payment-service/values.yaml:78]: value not set

File: appConfig/appConfig.snd1.json [hunk 3]
```diff
@@ -1,158 +1,170 @@
...
+    },
+    {
+        "key": "POSTGRES_SCHEMA_NAME",
+        "value": "public",
+        "label": "{{serviceName}}",
+        "content_type": "",
+        "tags": {}
+    },
+    {
+        "key": "APPINSIGHTS_INSTRUMENTATIONKEY",
```
Comment (kbabigumira) [inline appConfig/appConfig.snd1.json:73]: Remove this, it should not be checked in

File: helm/ffc-demo-payment-service/values.yaml (hunk 2 above)
Comment (jeevankuduvaravindran) [reply helm/ffc-demo-payment-service/values.yaml]: clean up done

**Review by DaRoszko — DISMISSED** [review]

**Review by ajaysoni-cap — APPROVED** [review]

**Pull request merged**